hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/OzoneRpcClientTests.java

```diff
@@ -1440,6 +1450,14 @@ private static void assertMetadataUnchanged(OzoneKeyDetails original, OzoneKeyDe
     assertEquals(original.getMetadata(), rewritten.getMetadata());
   }
 
+  private static void checkFeatureEnable() {
```

Comment:
I think we should make the function name clear
```suggestion
  private static void checkAtomicKeyRewriteFeatureEnable() {
```

Reply:
May be better to keep the name and add a parameter for the feature being checked.